### ISSUE TITLE
Ability for activity admins to delete their activities

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/iRecord#211]
Currently, activities cannot be deleted.  In the interim, can activities be deleted within the warehouse database?

**Comment by johnvanbreda** [comment]
Yes, we just need to set the deleted flag on the relevant record in the groups table. It might also be worth removing foreign keys for the group_id from the samples table and the related group_ids in the cache tables.

**Comment by kitenetter** [comment]
Currently admins can remove activities from public view by setting the date at which the Activity ends. Using this option perhaps makes it less important to remove foreign keys in other tables.

However, for centres and activities under an organisation, setting a finish date does not remove the activity from the main view when logged in as an activity admin, which can result in untidy and potentially misleading displays of activities that are no longer in use. It would be better if expired activities and centres could be moved to the "My finished activities" tab.

It would be useful for admins to be able to delete activities, as people often create one or two test activities before settling on a 'real' one. However, that would mean finding a way of dealing with any redundant foreign keys.
